scripts/configure_valhalla.sh

```diff
@@ -219,12 +219,33 @@ if [[ "${do_build}" == "True" ]]; then
     fi
     maybe_create_dir ${ELEVATION_PATH}
     echo ""
-    echo "================================="
+    echo "================================"
     echo "= Download the elevation tiles ="
-    echo "================================="
+    echo "================================"
     valhalla_build_elevation --from-tiles --decompress -c ${CONFIG_FILE} -v || exit 1
   fi
 
+fi
+
+# Use OSMSpeeds default_speeds
+
+updated_default_speed_config=False
+if [[ $use_default_speeds_config == "True" ]]; then
+  if test -f "${DEFAULT_SPEEDS_CONFIG}"; then
+    echo "WARNING: found default speed config, skipping download"
+  else 
+
+    echo ""
+    echo "======================================"
+    echo "= Download the default speeds config ="
+    echo "======================================"
+    download_file "${default_speeds_config_url}" "${DEFAULT_SPEEDS_CONFIG}"
+    updated_default_speed_config=True
+  fi
+  jq --arg d "${DEFAULT_SPEEDS_CONFIG}" '.mjolnir.default_speeds_config = $d' "${CONFIG_FILE}" | sponge "${CONFIG_FILE}"
+fi
+
+if [[ "${do_build}" == "True" ]] || [[ updated_default_speed_config == "True" ]]; then
```

Comment:
moved this out into additional if block so we also enhance if the config was newly downloaded.

Reply:
hm, we don't support starting/stopping from custom build stages, so for readability I'd keep this with the rest of those options further at the top and treat it the same as e.g. admins or so by setting `do_build`. we could also allow for `FORCE` to re-download and force a new tile build so it's more inline with the other options?

Reply:
other than that, looks good

Reply:
hm, on second thought you might have a point. you're basically saying, if you have a graph already, but didn't have a speeds json, we'll only enhance the graph but won't build it from scratch. actually that's fine. I guess the biggest gotcha will be that we won't tar the new graph if there was already one, we'd just leave the stuff in the tile dir. for that one would have to set `build_tar = Force`. but actually the same is true for any graph rebuild.